redis-sink/src/main/resources/application.yml

```diff
@@ -0,0 +1,9 @@
+server:
+  port: 0
+spring:
+  cloud:
+    stream:
+      bindings:
+        input: testtock
```

Comment:
Suggestions?  'Output' assuming binding would prefix with application name?  Need to circle back to the logsink as well with whatever we decide.